### PR TITLE
Add tensoboard integration

### PR DESCRIPTION
**Pull request by ZidongLiu**
image

### CODE REVIEW

File: cezo_fl/server.py [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 
 from cezo_fl.shared import update_model_given_seed_and_grad
 from shared.metrics import Metric, accuracy
 from gradient_estimators.random_gradient_estimator import RandomGradientEstimator as RGE
 
 
+class LocalUpdateResult(TypedDict):
+    grad_tensors: Sequence[torch.Tensor]
+    step_accuracy: float
+    step_loss: float
```
Comment (bichengying) [inline cezo_fl/server.py:15]: we typically prefer to use dataclasses

**Review by bichengying — APPROVED** [review]
LGTM

File: cezo_fl/server.py (hunk 1 above)
Comment (ZidongLiu) [reply cezo_fl/server.py:15]: let me look into that, never used that

Comment (ZidongLiu) [reply cezo_fl/server.py]: changed to dataclass

**Pull request merged**